packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -26,16 +26,27 @@ import {
 } from './TokenSearchUtils'
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
 import { useBalance } from '../../hooks/useBalance'
-import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import { ERC20BridgeToken, TokenType } from '../../hooks/arbTokenBridge.types'
 import { useTokenLists } from '../../hooks/useTokenLists'
 import { warningToast } from '../common/atoms/Toast'
 import { TokenRow } from './TokenRow'
+import { CommonAddress } from '../../util/CommonAddressUtils'
+import { ArbOneNativeUSDC } from '../../util/L2NativeUtils'
+import { isNetwork } from '../../util/networks'
+import { useUSDCWithdrawalConfirmationDialogStore } from './TransferPanel'
 
 enum Panel {
   TOKENS,
   LISTS
 }
 
+const ARB_ONE_NATIVE_USDC_TOKEN = {
+  listIds: new Set<number>(),
+  type: TokenType.ERC20,
+  l2Address: CommonAddress.ArbitrumOne.USDC,
+  ...ArbOneNativeUSDC
```

Comment:
NIT: Ideally, the spread operator object precedes the new keys so that they can over-write it if required.